### PR TITLE
Add more presentation Videos

### PR DESCRIPTION
**Pull request by jtattermusch**
- move videos from About section to docs/talks
- add a few more presentation videos

The endresult can be checked here: http://jtattermusch.github.io/grpc.github.io/docs/talks/

### CODE REVIEW

File: js/common.js [hunk 1]
```diff
@@ -9,8 +9,13 @@ var firstScriptTag = document.getElementsByTagName('script')[0];
 firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);
 
 // Array of videoIds
-// The key corresponds to the data attributes in about/index.html
+// The key corresponds to the data attributes in docs/talks/index.html
+// TODO(jtattermusch): is there a way to not require editing this when adding new videos?
 var playerInfoList = [
+    {type: 'yt', key: '5QH67PXNbaI'},
```
Comment (jtattermusch) [inline js/common.js]: btw, I think requiring this is pretty unfortunate, it makes it very hard to add new videos.

**Comment by jtattermusch** [comment]
@LisaFC  do you have any idea why the "tile layout" is broken in http://jtattermusch.github.io/grpc.github.io/docs/talks/?

**Comment by jtattermusch** [comment]
@LisaFC  friendly ping.

**Comment by hsaliak** [comment]
@LisaFC ping

**Comment by LisaFC** [comment]
Huh, not sure why the tiling is broken, I'll have a look. Have added the content in anyway while we fix the layout....

**Comment by jtattermusch** [comment]
@LisaFC  two videos also fail to load: https://grpc.io/docs/talks/  Not sure what's wrong but I'm pretty sure that wasn't the case back when I created the PR.

**Pull request merged**